.eslintrc.json

```diff
@@ -76,11 +76,6 @@
     // 7.10 https://github.com/airbnb/javascript#functions--constructor
     "no-new-func": "error",
 
-    // 7.11 https://github.com/airbnb/javascript#functions--signature-spacing
-    // 19.2 https://github.com/airbnb/javascript#whitespace--before-blocks
-    "space-before-function-paren": ["error", "never"],
-    "space-before-blocks": ["error", "always"],
```

Comment:
OOC, why remove these?

Reply:
Due to the fact that in the .js files in this test, we often use `async` methods that generate a lint problem, if the above rules are defined

<img width="554" alt="image" src="https://user-images.githubusercontent.com/5073455/223713079-d8302d1a-f09b-451b-8639-ad37cddca1a8.png">

Reply:
They only generate a lint error if there is a space before the parenthesis (as it should) :grin:

That said, I do agree that `async` arrow function look ugly without a space before the parenthesis. You can configure the rule to enforce that, though. See https://github.com/eslint/eslint/issues/13983#issuecomment-754598434.

Reply:
I restored the deleted rules and reconfigured them